app_legacy/Helpers/database/game.php

```diff
@@ -109,44 +109,24 @@ function getGameMetadata(
     $numAchievements = count($achievementDataOut);
 
     if (isset($user)) {
-        $query = "SELECT ach.ID, aw.Date, aw.HardcoreMode
-                  FROM Awarded AS aw
-                  LEFT JOIN Achievements AS ach ON ach.ID = aw.AchievementID
-                  WHERE ach.GameID = :gameId AND ach.Flags = :achievementType AND aw.User = :username";
```

Comment:
This query is now cached by `getUserAchievementUnlocksForGame`

Reply:
Note that this query isn't as expensive as I originally postulated. It's joining Awarded, not Activity.